tests/behavioural/features/steps/thread_get.py

```diff
@@ -35,9 +35,47 @@ def update_encrypted_jwt():
 
 headers['Authorization'] = update_encrypted_jwt()
 
-# Scenario: Respondent and internal user have a conversation and respondent retrieves the conversation
+
+# Scenario 1: Respondent and internal user have a conversation and respondent retrieves the conversation (see common steps)
+# Scenario 2: Respondent and internal user have a conversation and respondent retrieves the conversation (see common steps)
+# Scenario 3: Respondent and internal user have a conversation, including a draft, and respondent retrieves the conversation (see common steps)
+# Scenario 4: Respondent and internal user have a conversation, including a draft, and internal user retrieves the conversation
+@then("all messages from that conversation should be received including draft")
+def step_impl_assert_all_messages_from_conversation_are_received(context):
+    response = flask.json.loads(context.response.data)
+    nose.tools.assert_equal(len(response), 5)
+
+
+# Scenario 5: Respondent and internal user have a conversation and internal user retrieves that conversation from multiple conversations
+@given("internal user has multiple conversations")
+def step_impl_internal_user_has_multiple_conversations(context):
+
+    for _ in range(0, 2):
```

Comment:
I don't know why `_` is there, but `for x in range` is used other places, keep consistent?